R/dettl_runner.R

```diff
@@ -51,6 +51,9 @@ run_import <- function(import, run_stages = NULL, dry_run = FALSE) {
     stop(
       "Can only run import for non null data import with class 'DataImport'.")
   }
+  if (!dry_run && !git_is_clean(import$path)) {
```

Comment:
the git status check should probably be the root directory (i.e. with the main dettl config) not the import path itself, because that covers changes in things like the configuration itself (EDIT: though I now see that is handled by your status function, though that's not totally obvious from the call here)

Reply:
Should be more clear now function name has been updated.